hub/api/tests/test_dataset_handler.py

```diff
@@ -0,0 +1,59 @@
+from hub.util.exceptions import DatasetHandlerError
+from click.testing import CliRunner
+import pytest
+from hub import dataset
```

Comment:
```suggestion
import hub
```

i really don't like this convention, we should set the stage for everywhere at every level to only use `import hub`